src/test/resources/io/gravitee/policy/jws/cert-with-crl/README.md

```diff
@@ -0,0 +1,112 @@
+This folder contain only needed for testing stuff.
```

Comment:
Maybe we should add a link to this readme in the main one, or one in `src/test` because this one is a bit hidden by itself

Reply:
I don't know if links work in PS or if we really want to show it "publicly", but it's OK for me.

Reply:
yeah maybe only in `src/test`